ui-tests/src/test/java/org/openmrs/reference/page/HomePage.java

```diff
@@ -43,7 +44,7 @@ private boolean isAppButtonPresent(String appId) {
     }
 
     public String getLoggedUsername() {
-        return findElement(By.cssSelector(".identifier")).getText();
+        return findElement(LOGGED_IN_USER).getText();
```

Comment:
Inspection on the `element` in context shows there are more than one element using the current `value property`. The suggested` attribute value property` uniquely identifies the element thus overriding.

Reply:
> you may need to start from .identifier and check it's children to target not the icon but the label

I opted for a reliable `attribute property value` that may not easily break to ensure code sustainability and wrapped it into a commit with id `2421397`